www/src/app/actions/index.js

```diff
@@ -0,0 +1,88 @@
+import * as alf from '../services/alfred-api-service';
+
+export const types = {
+  // user
+  SET_AUTHOR:                        'SET_AUTHOR',
+
+  //ui
+  SET_EDITING:                       'SET_EDITING',
+  CANCEL_EDITING:                    'CANCEL_EDITING',
+
+  // updates
+  FETCH_UPDATES_BY_AUTHOR:           'FETCH_UPDATES_BY_AUTHOR',
+  FETCH_UPDATES_BY_AUTHOR_COMPLETED: 'FETCH_UPDATES_BY_AUTHOR_COMPLETED',
```

Comment:
I think the only one that then looks out of place a patch and post actions, do you have thoughts  on that?